app/models/miq_server/log_management.rb

```diff
@@ -155,7 +156,7 @@ def pg_log_patterns
     [pg_data.join("*.conf"), pg_data.join("pg_log/*")]
   end
 
-  def post_current_logs(taskid)
+  def post_current_logs(taskid, context)
```

Comment:
Maybe `context = 'MiqServer'` as a default coming into this method?  Otherwise, we might have to deal with existing queue items using the old signature.